### ISSUE TITLE
Add more generic way to expose a warning log if an accessible name is missing

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3330]
Scope
---
Add global validation warning to log warning messages if users don't provide an accessible name.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
Share any further information that is relevant for processing.

Acceptance criteria
---
- [ ] Criteria

Subtasks
---
- [ ] Task
